R/dots-ellipsis.R

```diff
@@ -55,16 +55,16 @@ check_dots <- function(env = caller_env(), error, action, call) {
   }
 
   proms <- ellipsis_dots(env)
-  used <- map_lgl(proms, promise_forced)
+  unused <- !map_lgl(proms, promise_forced)
 
-  unused <- names(proms)[!used]
   action_dots(
     error = error,
     action = action,
-    message = paste0(length(unused), " arguments in `...` were not used."),
-    dot_names = unused,
+    message = "Arguments in `...` must be used.",
```

Comment:
Maybe all arguments?

Reply:
I might be wrong but to me "All arguments" implies that some but not all of the arguments were used, which isn't a claim we can make here.